tests/fork/price-oracle.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { BigNumber } from "ethers";
+import { makeSuite, TestEnv } from "../utils/make-suite";
+import { expect } from "chai";
+
+makeSuite("Price Oracle", (testEnv: TestEnv) => {
+  const zero = BigNumber.from("0");
+
+  it("Check Assets price", async () => {
+    const { weth, dai, usdc, oracle } = testEnv;
+
+    if (!weth || !oracle) {
+      console.error("weth or oracle not found in testEnv");
+      return;
+    }
```

Comment:
You can use `expect` here to aset weth and oracle are defined, and maybe better to have a failing test if that's the case

Reply:
Or better `assert` to have custom message describing the error: https://www.chaijs.com/api/assert/